tasks/17_truncate-string/truncate-string.js

```diff
@@ -0,0 +1,12 @@
+export const truncateString = (str, num)=>{
+  let result = '';
+  if ( str.length > num) {
+    result = str.slice(0, num-3) +'...';
```

Comment:
what is  3 for in num-3?

Reply:
As the description goes: "Note that inserting the three dots to the end will add to the string length." If we want to pass tests we have to substract 3(number of dots) from num.